Doc/c-api/call.rst

```diff
@@ -340,10 +355,12 @@ please see individual documentation for details.
    Return the result of the call on success, or raise an exception and return
    *NULL* on failure.
 
+   This function is not part of the `limited API <stable>`_.
```

Comment:
Maybe put it also in the limited API. I don't see any issue with the stable ABI.